docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -87,7 +88,34 @@ CAST function converts the value returned from a table column to the specified d
 | `TIMESTAMP` | `DATE`, `TEXT` |
 | `TIMEUUID` | `DATE`, `TIMESTAMP` |
 
-### Examples
+## WriteTime function
+We can get the timestamp when the column/row was writen.
+Assuming we have a table `page_views` and a column named `views`:
```

Comment:
For example, suppose we have a table `page_views` with a column named `views`:

(add the For example prefix, after the more formal description ).